src/cargo/ops/cargo_compile.rs

```diff
@@ -182,7 +183,7 @@ pub fn compile_pkg<'a>(root_package: &Package,
         Some(args) => {
             if to_builds.len() == 1 {
                 let targets = try!(generate_targets(to_builds[0], profiles,
-                                                    mode, filter, release));
+                                                    mode, filter, all_feature, release));
```

Comment:
What should be done here? `to_builds[0].summary().features()` returns a `&HashMap<String, Vec<String>>`, which is a collection of features from the `[features]` section of `Cargo.toml`, but if I understand correctly all_feature should have a list of enabled features for the crate that's going to be built. I need some pointers, because I'm a bit lost.